install/kubernetes/cilium/charts/hubble-relay/values.yaml

```diff
@@ -1,11 +1,23 @@
+enabled: false
+
+podSecurityPolicy:
+  enabled: false
```

Comment:
nit: I think PSP is deprecated upstream, do we still need this?

Reply:
if it is deprecated, I can kill this

Reply:
Based on what I can see in the documentation, this is a current (Beta) feature that is currently supported https://kubernetes.io/docs/concepts/policy/pod-security-policy/

Reply:
@seanmwinn PSP will be removed from k8s so we won't support it https://github.com/cilium/cilium/pull/12469

Reply:
removed!